asm/istio/options/cni-managed.yaml

```diff
@@ -207,6 +279,46 @@ spec:
               value: "cni.istio.io/uninitialized"
             - name: "REPAIR_BROKEN-POD-LABEL-VALUE"
               value: "true"
+        - name: mdp-controller
+          # yamllint disable-line rule:line-length
+          image: "IMAGE_HUB/mdp:IMAGE_TAG"  # {"$ref":"#/definitions/io.k8s.cli.substitutions.gke-cni-image"}
+          command:
+            - mdp
+            - server
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+            privileged: false
+            readOnlyRootFilesystem: true
+            runAsGroup: 1337
```

Comment:
why do we need to run as 1337?

Reply:
is it just because of the base image?

Reply:
All istio container, including istiod, runs as 1337. I guess it is a convention..

Reply:
The base image has this uid, ok.